src/app/recipes/recipes.service.spec.ts

```diff
@@ -0,0 +1,49 @@
+import { TestBed } from '@angular/core/testing';
+import { RecipesService } from './recipes.service';
+import { Recipe } from './recipe.interface';
+
+describe('DataService', () => {
+  let service: RecipesService;
+  let testRecipeObject: Recipe = {
+    title: 'title',
+    description: 'description',
+    ingredientsList: [{ amount: 1, name: 'name', unit: 'unit' }],
+    instructions: ['instructions'],
+    numberOfPortions: 2,
+    id: 1,
+  };
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({});
+    service = TestBed.inject(RecipesService);
+  });
+
+  it('Service should be created', () => {
+    expect(service).toBeTruthy();
+  });
+
+  //are tests working?
+  it('should be something', () => {
+    expect(true).toBeTruthy();
+  });
+
+  //add recipe
+  it('should add one to length', () => {
```

Comment:
cases should be more descriptive, this is also a documentation of sorts

Reply:
"should increase the number or recipes in array by 1"

Reply:
Also check if added recipe has correct data coherence, there may be some mutation that would lead to bugs